examples/illustrations/Oresteia/Oresteia.json

```diff
@@ -1480,8 +1477,8 @@
             "uco-core:isDirectional": true,
             "uco-core:hasFacet": [
                 {
-                    "@type": "uco-observable:AttachmentFacet",
-                    "rdfs:comment": "TODO: uco-observable:Attachment seems ill-defined.",
+                    "@type": "xsd:string",
```

Comment:
This is wrong.  `xsd:string` is a OWL Datatype, so a JSON-LD dictionary with `"@type": "xsd:string"` is an RDF Literal, not an object.

The `...corpses.jpg` URL will have to be attached via an `ObservableRelationship` object, instead of this former `AttachmentFacet`.  Please make a new `ObservableRelationship` identified with a UUIDv4.